package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "flashfeetui",
+  "version": "1.0.0",
+  "description": "User Interface for team FlashFeet",
+  "main": "app.js",
+  "scripts": {
+    "start": "npx tsc && cp -R views dist && node dist/app.js",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint ."
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
```

Comment:
Similar to other PR's. Some dev dependencies in here. Should be move to devDependencies section.